src/cell/main.c

```diff
@@ -86,6 +89,124 @@ main(int argc, char* argv[])
 	return retval;
 }
 
+static bool
+generate_project(char* path)
+{
+	char* project_name;
+	char* author;
+	char* width;
+	char* height;
+	char* summary;
+	path_t* app_path;
+	path_t* project_path;
+	path_t* template_path;
+	path_t* tmpl_path;
+	path_t* mjs_path;
+	FILE* mjs_file = NULL;
+	char* tmpl_str;
+	char* mjs_str;
+	bool success = false;
+
+	printf("Enter game name: ");
+	project_name = malloc(256);
```

Comment:
Okay so I know you're not super-experienced with C, but I'm going to have to call you out for doing this regardless. :)  Since you're unconditionally allocating a fixed 256 bytes and never resizing the array you could just declare the variables like:
```c
char project_name[256];
```
Doing so would allocate the buffer on the stack, which both avoids the need to free it as well as losing the overhead of a dynamic allocation.  `malloc()` isn't free; stack allocation typically is, which is why I now use `alloca()` when dynamically allocating small temporary buffers. (not that the performance issue is at all relevant for a one-time operation like this but for future reference...)